components/publish/PublishWidgetList.vue

```diff
@@ -20,9 +20,8 @@ const {
   dialogLabelledBy?: string
 }>()
 
-const threadItems = computed(() =>
-  useThreadComposer(draftKey, initial).threadItems.value,
-)
+const threadComposer = useThreadComposer(draftKey, initial)
+const threadItems = computed(() => threadComposer.threadItems.value)
```

Comment:
I still don't understand why this should be a computed, Ig uess it can be used with `threadComposer.threadItems.value` in the template